ilert_nagios.py

```diff
@@ -120,20 +125,33 @@ def create_xml(apikey, payload):
     return xml_doc
 
 
+def str_to_bool(v):
+    if isinstance(v, bool):
+       return v
+    if v.lower() in ('yes', 'true', 't', 'y', '1'):
+        return True
+    elif v.lower() in ('no', 'false', 'f', 'n', '0'):
+        return False
+    else:
+        raise argparse.ArgumentTypeError('Boolean value expected.')
+
 def main():
     parser = argparse.ArgumentParser(description='send events from Nagios (and its forks, such as Icinga) to iLert')
     parser.add_argument('-m', '--mode', choices=['nagios', 'save', 'cron', 'send'], required=True,
                         help='Execution mode: "save" persists an event to disk and "send" submits all saved events '
                              'to iLert. Note that after every "save" "send" will also be called.')
     parser.add_argument('-a', '--apikey', help='API key for the alert source in iLert')
-    parser.add_argument('-e', '--endpoint', default='https://ilertnow.com',
+    parser.add_argument('-e', '--endpoint', default='https://api.ilert.com',
```

Comment:
because you're changing the default end point, make sure to include a *What's new* section in the README. Customers probably need to add api.ilert.com to their prioxy config.

Reply:
ok